### DISCUSSION TITLE
有没有针对小程序使用F2api的文档？我搜索很久 并没有找到。

### DISCUSSION THREAD

**Discussion by ShiShanLing** [antvis/G2#4964] [Q&A] [answered]
- [ ] I have searched the [issues](https://github.com/antvis/g2/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
可以让我在小程序中更加顺畅的使用F2

### What does the proposed API look like?
我期待有一个针对小程序中使用F2的开发文档，目前只有一个案例 我在使用其他案例是手足无措

**Comment by hustcc** [comment, marked as answer]
@ShiShanLing 可以看看 F2 官网教程，我看了下，有一篇文档[介绍如何在小程序中使用](https://f2.antv.antgroup.com/tutorial/framework/miniprogram)。

**Comment by ShiShanLing** [comment]
嗯感谢你的回答，在你回答的链接中我看到了使用方法。
